e2e/cypress/integration/scroll/fixed_width_spec.js

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+const timeouts = require('../../fixtures/timeouts');
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @scroll
+
+describe('Scroll', () => {
+    let testTeam;
+    let testChannel;
+
+    before(() => {
+        cy.apiUpdateConfig({
+            ServiceSettings: {
+                EnableLinkPreviews: true,
+            },
+        });
+
+        cy.apiInitSetup().then(({team, channel}) => {
+            testTeam = team;
+            testChannel = channel;
+            cy.visit(`/${testTeam.name}/channels/${channel.name}`);
+        });
+    });
+
+    it('MM-T2368 Fixed width', () => {
+        const link = 'https://www.bbc.com/news/uk-wales-45142614';
+        const gifLink = '![gif](http://i.giphy.com/xNrM4cGJ8u3ao.gif)';
+        cy.postMessage('This is the first post');
+        cy.postMessage(link);
+        cy.postMessage(gifLink);
+
+        const commonTypeFiles = ['jpg-image-file.jpg', 'gif-image-file.gif', 'mp3-audio-file.mp3', 'mpeg-video-file.mpg'];
+        commonTypeFiles.forEach((file) => {
+            cy.get('#fileUploadInput').attachFile(file);
+            cy.postMessage(`Attached with ${file}`);
+        });
+        cy.postMessage('This is the last post');
+        getUserNameTitle().eq(0).invoke('height').as('initialUserNameHeight');
+        getThumbnailPost().eq(0).invoke('height').as('initialThumbnailHeight');
+        getAttachmentPost().eq(0).invoke('height').as('initialAttachmentHeight');
+        getInlinImgPost().eq(0).invoke('height').as('initialInlineImgHeight');
+        getFirstTextPost().eq(0).invoke('height').as('initial1stPostHeight');
```

Comment:
nit: for consistency with "last"
```suggestion
        getFirstTextPost().eq(0).invoke('height').as('initialFirstPostHeight');
```

Reply:
:heavy_check_mark: